Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -203,7 +204,7 @@ private boolean startExport(Process process, LegacyMetsModsDigitalDocumentHelper
 
     private boolean exportCompletedChildren(List<Process> children) throws DataException {
         for (Process child:children) {
-            if (processService.getProgress(child.getTasks(), null).equals(COMPLETED) && !child.isExported()) {
+            if (ProcessConverter.getCombinedProgressAsString(child, false).equals(COMPLETED) && !child.isExported()) {
```

Comment:
Wouldn't calling `ProcessConverter.getCombinedProgressAsString` with second parameter `considerChildren = false` with an parent process without own tasks always result in "000000000100", e.g. 100% locked (as oposed to the the expected "100000000000", e.g. 100% completed?

Reply:
If this list of children contains processes that have no tasks, they are not catched by this if statement, correct. I don't know what this export code is doing, which is why I didn't change the implementation of how the progress status is calculated.